api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -418,12 +417,12 @@ public void getCohortsContainingPatient_shouldNotReturnVoidedCohorts() throws Ex
 		assertFalse(service.getCohort(2).isVoided());
 		
 		// add a patient to both cohorts
-		Patient patientToAdd = new Patient(4);
+		Patient patientToAdd = new Patient(7);
```

Comment:
I was getting the referential integrity constraint for patient id 4. As you mentioned, there wasn't an instance of patient id 4 in the test data.